Svc/ComQueue/ComQueue.cpp

```diff
@@ -71,7 +71,9 @@ void ComQueue::configure(QueueConfigurationTable queueConfig,
         for (NATIVE_UINT_TYPE entryIndex = 0; entryIndex < FW_NUM_ARRAY_ELEMENTS(queueConfig.entries); entryIndex++) {
             // Check for valid configuration entry
             FW_ASSERT(queueConfig.entries[entryIndex].priority < TOTAL_PORT_COUNT,
-                      queueConfig.entries[entryIndex].priority, TOTAL_PORT_COUNT, entryIndex);
+                      queueConfig.entries[entryIndex].priority,
+                      TOTAL_PORT_COUNT,
```

Comment:
Cast

Reply:
Just to make sure, do you want to have the cast  `static_cast<FwAssertArgType>()` even if it is a compatible/equivalent type?

Reply:
Fixed